server/Dockerfile

```diff
@@ -65,6 +65,8 @@ RUN <<EOF
     ls -Rla /out/libs-root
 EOF
 
+RUN ls /usr/src/kanidm/target/release/kanidmd
```

Comment:
🤔 what's this here for?

Reply:
Opps, me trying to debug why the build was failing. I'll remove this.